crates/table/src/bflatn_to.rs

```diff
@@ -168,49 +168,73 @@ impl BflatnSerializedRowBuffer<'_> {
         let ty_alignment = ty.align();
         self.curr_offset = align_to(self.curr_offset, ty_alignment);
 
-        match (ty, val) {
```

Comment:
This change didn't really make much of a difference. It can probably be ignored. This function is definitely slow though and takes up quite a lot of time.